wemake_python_styleguide/constants.py

```diff
@@ -359,3 +359,9 @@
 # Used to specify a pattern which checks variables and modules for underscored
 # numbers in their names:
 UNDERSCORED_NUMBER_PATTERN: Final = re.compile(r'.+\D\_\d+(\D|$)')
+
+# List of functions in which parameters must be of type tuple instead
+# another iterables
+TUPLE_ARGUMENTS_METHODS = frozenset((
```

Comment:
This is a public constant. It should be moved to the upper part of the file and documented using `#:`